docs/modules/ROOT/pages/ec_sigstore_verify_attestation.adoc

```diff
@@ -4,12 +4,12 @@ Use sigstore to verify the attestation of an image.
 
 == Usage
 
-  result = ec.sigstore.verify_attestation(ref: string, opts: object<certificate_identity: string, certificate_identity_regexp: string, certificate_oidc_issuer: string, certificate_oidc_issuer_regexp: string, ignore_rekor: boolean, public_key: string, rekor_url: string>)
+  result = ec.sigstore.verify_attestation(ref: string, opts: object<ignore_rekor: boolean>[string: string])
```

Comment:
Not sure if it's intentional to keep just one, but should it be this, since the keys for `opts` are documented below?

```suggestion
  result = ec.sigstore.verify_attestation(ref: string, opts: object[string: string])
```

Reply:
Oh, maybe it's generated, in which case disregard..?

Reply:
Yeah. It's generated :)